### ISSUE TITLE
playlist videos

### ISSUE THREAD

**Issue by Vernityz** [talmobi/yt-search#35]
can you add video duration to playlist videos?

```js
    {
      title: '',
      videoId: '',
      listId: '',
      thumbnail: '',
      author: [Object]
    },
```

that will be great if u add a duration object.

**Comment by talmobi** [comment]
hello there,

published new version: `yt-search@2.5.0` with this addition, please give it a try~

Thanks for the suggsetion

**Comment by Vernityz** [comment]
Thank you, you are the best :)
